test/ctia/auth/jwt_test.clj

```diff
@@ -47,16 +47,6 @@
                                                        (sut/casebook-root-scope)})))
       "with all scopes you should have most capabilities except some very
        specific ones")
-  (is (= #{:search-casebook
-           :list-casebooks
-           :create-casebook
-           :read-casebook
-           :delete-casebook}
-         (set/difference
-          (sut/scopes-to-capabilities #{(sut/casebook-root-scope)})
-          (sut/scopes-to-capabilities #{(sut/entity-root-scope)})))
-      "entity scope don't provide casebook")
-
```

Comment:
Would you mind to replace that test to check that `(sut/scopes-to-capabilities #{(sut/casebook-root-scope)})` is a subset of `(sut/scopes-to-capabilities #{(sut/entity-root-scope)}))`?